examples/wip/moco/train_moco.py

```diff
@@ -141,8 +141,9 @@ def create_model(key, batch_size, image_size, model_def):
   input_shape = (batch_size, image_size, image_size, 3)
   with flax.nn.stateful() as init_state:
     with flax.nn.stochastic(jax.random.PRNGKey(0)):
-      (_, _), model = model_def.create_by_shape(
+      (_, _), initial_params = model_def.initby_shape(
```

Comment:
Typo

Reply:
oh, sorry for that! good that you caught it.